flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/pipeline/Predictor.scala

```diff
@@ -121,7 +121,7 @@ object Predictor {
 
         input.mapWithBcVariable(model){
           (element, model) => {
-            (element, predictOperation.predict(element, model))
+            (element, predictOperation.predict(element, model, resultingParameters))
```

Comment:
Hmm I'm not so sure either what is the best way to go here. My initial idea was to isolate the predict operation as much as possible from the code necessary to do the setup. I don't like the `ParameterMap` much, because it allows you to have access to much more information than necessary. IMO it's better to extract the information once at one place and then have an explicit representation of the information.